### ISSUE TITLE
Builds are not up to date

### ISSUE THREAD

**Issue by gahia** [adrienpoly/stimulus-flatpickr#60]
Hi!

It seems the builds are not up to date. I've been trying to use the `monthSelectorType` option with no luck until I've realized that the builds don't include these changes yet.

Not a big problem as the sources can be imported instead, but it's a bit confusing.

Thanks so much for the controller, it's just amazing :)

**Comment by adrienpoly** [comment]
Hum strange are you using 1.3? I thought I released 1.3 after this addition. If not I'll release a new package

**Comment by gahia** [comment]
It seems so...

I've installed it with:

```
$ yarn add flatpickr
$ yarn add stimulus-flatpickr
```

and the result is:

```
└─ stimulus-flatpickr@1.3.0
└─ flatpickr@4.6.6
```

After couldn't find were the issue was, I've look for the `monthSelectorType` option in the webpacker builds but wasn't there, then is when I figure out the builds didn't include those changes. Importing the sources works perfect.

**Comment by adrienpoly** [comment]
Thanks for reporting this issue the builds where indeed not the latest! 
I have released the version 1.3.2 with the lastest build.

Here is a codesandbox with the monthSelectorType option
https://codesandbox.io/s/month-selector-type-2egrq?file=/index.html
